fda/grid.py

```diff
@@ -112,7 +349,8 @@ class FDataGrid:
 
     def __init__(self, data_matrix, sample_points=None,
                  sample_range=None, dataset_label='Data set',
-                 axes_labels=None):
+                 axes_labels=None, interpolation_order=1,
```

Comment:
I think that using parameters `interpolation_order`, `smoothness` and `monotone` we are assuming that the interpolation is always done with splines, is that right? I am not sure if that is the only interpolation that make sense. I think that it would be better a single parameter `interpolation` that receives an object with an "interpolator API". That would also simplify the getter and setter.

Reply:
Changed in 8c58331.